sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -358,7 +337,7 @@ def test_list_configuration_settings_multi_pages(self):
             pass
 
     def test_list_configuration_settings_null_label(self):
-        items = self.app_config_client.list_configuration_settings(labels=[""])
+        items = self.app_config_client.list_configuration_settings(label_filter="\0")
```

Comment:
Doesn't seem right, you used to have code code to replace an empty string with "\0" so customer never see "\0", why not keeping it?

Reply:
Yes. We used to escape the char for the user and in the new design, we will let user escape it by themselves. This is a change by design.